### ISSUE TITLE
Why Tampermonkey includes '@grant none' into each new user script?

### ISSUE THREAD

**Issue by john-cj** [Tampermonkey/tampermonkey#1537]
When you create a new script, it will have `@grant none` as a part of its preample. As far as I know, `@grant none`: ...

* not just disables GM_* functions, the unsafeWindow object, and some window functions,
* but also disables the sandbox, so that the script will run directly on the page itself, which means it will be very easy for a website to mess with such a script somehow, and typically we don't want this.

In other words, `@grant none` seems to do more "bad" things than "good" things, and so why Tampermonkey includes it by default into each new user script? Isn't it should be changed probably?

**Comment by derjanb** [comment]
Without special permissions, there is nothing for a site to "steal".

> so that the script will run directly on the page itself,

Scripts alway run in page context [unless you use TM BETA for Firefox](https://github.com/Tampermonkey/tampermonkey/issues/1186#issuecomment-1151485354) or set Sandbox mode at TM 4.18+ to force DOM mode, which has several disadvantages.
